packages/cache/src/internal/cacheHttpClient.ts

```diff
@@ -31,12 +31,7 @@ import {
 const versionSalt = '1.0'
 
 function getCacheApiUrl(resource: string): string {
-  // Ideally we just use ACTIONS_CACHE_URL
-  const baseUrl: string = (
-    process.env['ACTIONS_CACHE_URL'] ||
-    process.env['ACTIONS_RUNTIME_URL'] ||
-    ''
-  ).replace('pipelines', 'artifactcache')
+  const baseUrl: string = process.env['ACTIONS_CACHE_URL'] || ''
```

Comment:
Do we need `|| ''`?

Reply:
Actually we don't need it but safeguard was there from beginning so i didn't remove it